### DISCUSSION TITLE
Make an option to have Davinci as a chat bot. (for those who are ready to pay)

### DISCUSSION THREAD

**Discussion by nblxba** [iTzArshia/GPT-Discord-Bot#6] [Ideas]
Hey, iTzArshia!
Wonderful bot.

I'm struggling to make Davinci as a default chat bot. Cause it is huge in conversation. Chat turbo is too simple.
Got no success so far.
As I can see, accodring to your skills, you can do it like in a 5 minutes.

My idea is to have a dialogue  in "utils\prompts\chatCompletion.txt". and each question and answer from discord should be added to the next prompt. So the bot could follow.
Can you make it in configs\chatbot2.js file?
I would greatly appreciate.

**Comment by iTzArshia** [comment]
Well, I have to change the whole code for that but sure I will update it but don't know when! but I will add an option in the chatbot config file to set the model for the chatbot.

Also it will remember the conversations you can try it say your name to it and then ask whats your name

**Reply by nblxba** [reply]
Would be great. So far it is the best bot on Java for discord.
